examples/spark/spark-ml-pipe-lgb.py

```diff
@@ -49,23 +49,22 @@
     }
 
     with log_exec_time():
-        train_df, test_df = prepare_test_and_train(spark, path, seed)
+        train_df, test_df = prepare_test_and_train(dataset, seed)
 
-        task = SparkTask(task_type)
+        task = SparkTask(dataset.task_type)
         score = task.get_dataset_metric()
         sreader = SparkToSparkReader(task=task, cv=cv, advanced_roles=False)
         spark_ml_algo = SparkBoostLGBM(freeze_defaults=False)
         spark_features_pipeline = SparkLGBAdvancedPipeline(**ml_alg_kwargs)
         spark_selector = SparkSelectionPipelineWrapper(
             BugFixSelectionPipelineWrapper(ImportanceCutoffSelector(
-                cutoff=0.0,
                 feature_pipeline=SparkLGBSimpleFeatures(),
                 ml_algo=SparkBoostLGBM(freeze_defaults=False),
                 imp_estimator=ModelBasedImportanceEstimator()
             ))
         )
 
-        sdataset = sreader.fit_read(train_df, roles=roles, persistence_manager=persistence_manager)
+        sdataset = sreader.fit_read(train_df, roles=dataset.roles, persistence_manager=persistence_manager)
```

Comment:
the [same](https://github.com/sb-ai-lab/SLAMA/pull/7/files#r1223141108)

Reply:
These interfaces follows interfaces of LightAutoML, which is an "origin" of SLAMA.
Not sure if it is a good idea to change these intrefaces. Need a discussion on this matter. 
(though SLAMA change them if it is necessary for performance purposes or for convenience related to Spark specific)